images/runtime/node/22/bookworm.Dockerfile

```diff
@@ -0,0 +1,81 @@
+ARG BASE_IMAGE
+
+# Startup script generator
+FROM mcr.microsoft.com/oss/go/microsoft/golang:1.23.1-bookworm as startupCmdGen
+
+# GOPATH is set to "/go" in the base image
+WORKDIR /go/src
+COPY src/startupscriptgenerator/src .
+ARG GIT_COMMIT=unspecified
+ARG BUILD_NUMBER=unspecified
+ARG RELEASE_TAG_NAME=unspecified
+ENV RELEASE_TAG_NAME=${RELEASE_TAG_NAME}
+ENV GIT_COMMIT=${GIT_COMMIT}
+ENV BUILD_NUMBER=${BUILD_NUMBER}
+RUN chmod +x build.sh && ./build.sh node /opt/startupcmdgen/startupcmdgen
+
+#FROM oryxdevmcr.azurecr.io/private/oryx/oryx-node-run-base-bookworm:${BUILD_NUMBER}
+FROM ${BASE_IMAGE}
+
+RUN groupadd --gid 1000 node \
+  && useradd --uid 1000 --gid node --shell /bin/bash --create-home node
+
+RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
+  && case "${dpkgArch##*-}" in \
+    amd64) ARCH='x64';; \
+    ppc64el) ARCH='ppc64le';; \
+    s390x) ARCH='s390x';; \
+    arm64) ARCH='arm64';; \
+    armhf) ARCH='armv7l';; \
+    i386) ARCH='x86';; \
+    *) echo "unsupported architecture"; exit 1 ;; \
+  esac
+
+ARG NODE22_VERSION
+ENV NODE_VERSION ${NODE22_VERSION}
+ENV NPM_CONFIG_LOGLEVEL info
+ARG BUILD_DIR=/tmp/oryx/build
+ARG IMAGES_DIR=/tmp/oryx/images
+
+COPY nodejs-bookworm-${NODE22_VERSION}.tar.gz .
+RUN set -e \
+    && mkdir -p /opt/nodejs/${NODE22_VERSION} \
+    && tar -xzf nodejs-bookworm-${NODE22_VERSION}.tar.gz -C /usr/local \
+    && rm nodejs-bookworm-${NODE22_VERSION}.tar.gz \
+    && ln -s /usr/local/bin/node /usr/local/bin/nodejs
+
+ARG NPM_VERSION
```

Comment:
what NPM and PM2 version are we adding for this? Have we verified that we are adding the minimum needed npm version for node22

Reply:
NPM 10.7.0 version and PM2 5.3.1 version. As discussed, we will upgrade npm version once node 22 becomes LTS